src/WebJobs.Extensions.DurableTask/ContextImplementations/RemoteOrchestratorContext.cs

```diff
@@ -182,5 +185,20 @@ private void SetResultInternal(OrchestratorExecutionResult result)
 
             this.executionResult = result;
         }
+
+        private void ValidateCustomStatusSize(string customStatus)
+        {
+            // Azure Table Storage enforces a 32 KB limit if a property value is a UTF-16 encoded string.
+            // We apply a 16 KB limit here to align with our in-process model.
+            const int MaxCustomStatusSizeInKB = 16;
+            double customStatusSizeInKB = customStatus != null ? Encoding.Unicode.GetByteCount(customStatus) / 1024.0 : 0;
```

Comment:
Not a big deal, but do we really need floating-point computation here? Consider defining the constant as `16*1024` and just comparing it with the `GetByteCount` result. This _may_ be more performant (though the impact is probably negligible) but, more importantly, this will eliminate some mental overhead for the reader of this code.